packages/hardhat/hardhat.config.ts

```diff
@@ -12,7 +12,7 @@ import "hardhat-deploy-ethers";
 
 // If not set, it uses ours Alchemy's default API key.
 // You can get your own at https://dashboard.alchemyapi.io
-const providerApiKey = process.env.ALCHEMY_API_KEY || "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF";
+const providerApiKey = process.env.ALCHEMY_API_KEY || "";
```

Comment:
Do we want to keep this? It seems it's only being used when mainnet forking. Maybe

Reply:
Remove ALCHEMY_API_KEY from `.env.exmaple` itself and added FORKING_URL variable there so people can use any RPC url they want? Didn't set the default FORK_URL value to `https://cloudflare-eth.com` because if people run `yarn fork`  then `https://cloudflare-eth.com` will crash and also block that person's IP and it will affect frontend (since in frontend we are using same RPC URL for ENS)